plugins/modules/xattr.py

```diff
@@ -46,26 +44,25 @@
   state:
     description:
       - defines which state you want to do.
-        V(read) retrieves the current value for a O(key) (default)
-        V(present) sets O(path) to O(value), default if value is set
-        V(all) dumps all data
-        V(keys) retrieves all keys
-        V(absent) deletes the key
+      - V(read) retrieves the current value for a O(key).
+      - V(present) sets O(path) to O(value), default if value is set.
+      - V(all) dumps all data.
+      - V(keys) retrieves all keys.
```

Comment:
`read`, `all`, and `keys` sound like "states" we should deprecate once we have a replacement.

Reply:
raised issue #9317